common/src/main/java/com/microsoft/identity/common/internal/broker/BrokerErrorResponse.java

```diff
@@ -52,6 +52,16 @@ public BrokerErrorResponse(){
 
     }
 
+    public BrokerErrorResponse(MicrosoftTokenErrorResponse microsoftTokenErrorResponse){
```

Comment:
Minor nit, param could be `final` -- looks like `SchemaUtil` could use to have a formatter run over it too. Rest LGTM